src/WireMock.Net.Abstractions/Admin/Settings/ProxyAndRecordSettingsModel.cs

```diff
@@ -73,4 +73,9 @@ public class ProxyAndRecordSettingsModel
     /// Defines the Replace Settings
     /// </summary>
     public ProxyUrlReplaceSettingsModel? ReplaceSettings { get; set; }
+
+    /// <summary>
+    /// Proxy all Api calls, irrespective of any condition
+    /// </summary>
+    public bool ProxyAll { get; set; } = false;

```

Comment:
This new field should also be parsed like it's done here:
WireMockServerSettingsParser.cs

private static void ParseProxyAndRecordSettings(WireMockServerSettings settings, SimpleSettingsParser parser)